c/aws-c-common/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:18.04
+
+RUN apt-get -y update
+RUN apt-get -y upgrade
+
+RUN apt-get -y install git
+RUN apt-get -y install gcc-multilib
+
+RUN git clone https://github.com/gernst/sv-benchmarks.git
+RUN git clone https://github.com/gernst/aws-c-common.git
+
+WORKDIR /sv-benchmarks
+RUN git checkout aws-c-common
```

Comment:
Why is this necessary?

Reply:
The Dockerfile is supposed to be there for convenience, documentation, and the possibility to easily re-create the files.
Thanks for the feedback. I'll squeeze in the corrections today.